tests/integration/test.bulk_docs.js

```diff
@@ -958,5 +954,44 @@ adapters.forEach(function (adapter) {
       });
     });
 
+    it('2839 implement revs_limit', function (done) {
+
+      // We only implement revs_limit locally
+      if (adapter === 'http') {
+        return done();
+      }
+
+      var LIMIT = 50;
+      var db = new PouchDB(dbs.name, {revs_limit: LIMIT});
```

Comment:
Nice, I like having it as a constructor option. It needs to be passed to the server in `http/index.js` too, though, doesn't it? Also needs docs.